### ISSUE TITLE
Track collected treasures

### ISSUE THREAD

**Issue by MCilento93** [simenon/LostTreasure#70]
Dear developers, 
I have noticed there is no achievement to track the treasures collected. So one cannot know which are left or if already taken (when buying a new treasure map for example). 

Is there a way to empowering this addon to supply this lack of the game? I mean add an achievement ... or add a menu window with all treasures taken/left for zone. 
On the net there is no addon better than this on treasure maps.
Bests,

**Comment by Gamer1986PAN** [comment]
Hey, you can use the BeamMeUp and their Treasure Tab to view all your Treasure Maps and Survey Maps that you have in Inventory and even in the bank. For your current zone and all the others.

If you set your Lost Treasure to "all in Inventory" and set the correct timer (to your likings) to remove the "X" that is all you really need.

**Comment by Gamer1986PAN** [comment]
Looks like that:
![grafik](https://user-images.githubusercontent.com/44592199/218321440-ba3e5831-61df-482d-a105-d0667765f159.png)

**Comment by Gamer1986PAN** [comment]
If you mean you just want to track if there is a way to see if you have digged out a treasure map at least once - there used to be an option for this but any game update from ESO did manage to break that and i don't think anybody is willing to fix it, since you will have a time in your live where you know most of the spots in the back of your head anyway.

**Comment by MCilento93** [comment]
> Hey, you can use the BeamMeUp and their Treasure Tab to view all your Treasure Maps and Survey Maps that you have in Inventory and even in the bank. For your current zone and all the others.
> 
> If you set your Lost Treasure to "all in Inventory" and set the correct timer (to your likings) to remove the "X" that is all you really need.

I don't fully get your words at the end of this message. Now I have that option activated (show only treasure in the inventory). But once discovered I'll delete the map the track will be lost

**Comment by Gamer1986PAN** [comment]
Yeah the "X" vanishes once you collected your map. But as soon as you get the same map in the inventory again or have multible survey maps on the same spot the "X" does not dissapear or reappears if you take it out of the bank.

**Comment by MCilento93** [comment]
> Yeah the "X" vanishes once you collected your map. But as soon as you get the same map in the inventory again or have multible survey maps on the same spot the "X" does not dissapear or reappears if you take it out of the bank.

Understood.
So I think envigasing to develop and to introduce a sort of achievement of found treasures. 
For sure the dynamic of the game will remain the same, allowing the player to get multiple times the same treasure. 
But I think it would be great to add in the Journal session of the menu the treasures already found (starting from the addon is installed). Do you think that this is easy to include in your addon? Do you think interesting to develop? I read from the web  that many player claim the lack of this feature of the original game.

**Comment by Gamer1986PAN** [comment]
I guess it would be doable in any way to unlock LibTreasure entrys if you dig for a map the first time but i am not able to.
With that option you could hide the "X" as long as you did not dig it out at least once, that would force you to find the map at least once to unlock the "X".

If you find someone that is adding that option to Lost Treasure i am willing to upload it to the official Lost Treasure Channel once i tested it.
But i guess there wont be any volunteer for an option like that.
